lib/main_common.dart

```diff
@@ -88,8 +89,11 @@ Future<void> _firebaseInitializer(FlavorName flavorName) async {
 }
 
 void _activateManagers() {
+  ConnectionCheckStatusManager connectionCheckStatusManager =
+      ConnectionCheckStatusManager();
+  connectionCheckStatusManager.activateConnectionManager();
```

Comment:
Not initialize ConnectionCheckStatusManager but use ConnectionCheckStatusManager from Dependency Injection?